### PR TITLE
Add Picard into the list of projects

### LINKED ISSUE #57: Add Picard into the list of projects

### PR DESCRIPTION
**Pull request by tsukanov**
Not sure how I forgot it in the first place. :confounded:

*Fixes #57.*

### CODE REVIEW

**Comment by tsukanov** [comment]
This is how it looks like:
Screenshot from the home page

File: metabrainz/templates/index/index.html [hunk 1]
```diff
@@ -23,12 +23,17 @@
     </div>
 
     <div id="projects" class="row">
-      <div class="col-sm-6 with-img">
+      <div class="col-sm-offset-3 col-sm-6 with-img">
```
Comment (tsukanov) [inline metabrainz/templates/index/index.html]: `col-sm-offset-3` is for centering the block.

**Comment by 96187** [comment]
Have we decided that Picard is officially a MetaBrainz project now then? It's been kinda in limbo for a long time and isn't even under the metabrainz user thing here on GitHub.

**Comment by mayhem** [comment]
:+1:

**Pull request merged**